harness/features/multithreading.local.test.ts

```diff
@@ -380,6 +380,7 @@ describe('Multithreading Tests', () => {
                     etag: null,
                     rayId: null,
                     lastModified: null,
+                    skipSDKConfigEvent: true
```

Comment:
what is this for?

Reply:
The default check should not expect an sdkconfig event - the delay is set to 2 seconds - when the flush is 500ms - so it will never get an sdk config event before it flushes